compile.py

```diff
@@ -0,0 +1,67 @@
+import yaml
+
+def writeGroup(data, key):
+    contact = data["groups"][key]["contact"]
+    contribution = data["groups"][key]["contribution"]
+    members = data["groups"][key]["members"]
+    writer.write(f'\n\n')
+    writer.write(f'**{key}:** {contact}\n\n')
+    writer.write(f'*{contribution}*\n\n')
+
+    for name in members:
+        writer.write(f'- {name}\n')
+
+
+def writeGroupMarkdown(data, key):
+    contact = data["groups"][key]["contact"]
+    contribution = data["groups"][key]["contribution"]
+    members = data["groups"][key]["members"]
+
+    writer.write(f'\n\n')
+    writer.write(f'**{key}:** {contact}\n\n')
+    writer.write(f'*{contribution}*\n\n')
+
+    for name in members:
+        writer.write(f'- {name}\n')
```

Comment:
This makes the document really long. Maybe write the names separated by commas instead?